packages/luna/src/components/Lessons/RateLesson/RateLesson.tsx

```diff
@@ -0,0 +1,122 @@
+import { Dialog } from "@/components/Dialog/V2";
+import { Typography } from "@/components/Typography";
+import { useFormatMessage } from "@/hooks/intl";
+import { Void } from "@litespace/types";
+import React, { useCallback, useState } from "react";
+import Rate from "@litespace/assets/Rate";
+import { RatingStars } from "@/components/RatingStars";
+import { Button } from "@/components/Button";
+import { Textarea } from "@/components/Textarea";
+
+export const RateLesson: React.FC<{
+  close: Void;
+  tutorName: string | null;
+  studentId: number;
+  rateLoading: boolean;
+  initialRating?: number;
+  /**
+   * whether you are rating session or the platform itself
+   */
+  type: "session" | "platform";
+  onRate: ({
+    id,
+    value,
+    feedback,
+  }: {
+    /**
+     * ratee id (student)
+     */
+    id: number;
+    value: number;
+    feedback: string | null;
+  }) => void;
+}> = ({
+  close,
+  tutorName,
+  onRate,
+  studentId,
+  rateLoading,
+  type = "session",
+  initialRating,
+}) => {
+  const intl = useFormatMessage();
+  const [rating, setRating] = useState<number>(initialRating || 5);
+  const [feedback, setFeedback] = useState<string>("");
+
+  const onFeedbackchange = useCallback(
+    (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+      setFeedback(event.target.value);
+    },
+    []
+  );
+
+  const sendRating = useCallback(() => {
```

Comment:
`send` will make the imperssion that you are sending it somewhere. This is not the case in your code. I think `setRating` is a better name.